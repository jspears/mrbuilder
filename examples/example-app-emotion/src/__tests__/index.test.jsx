import React from 'react';
import {create} from 'react-test-renderer';
import App from '../index';
test('should render index', function(){
    expect(create(<App/>)).toMatchSnapshot();
})