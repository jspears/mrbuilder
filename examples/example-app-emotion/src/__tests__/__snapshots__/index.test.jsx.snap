// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render index 1`] = `
<div>
  <button
    className="css-17tckwd-Button e190i1mp0"
  >
    This my "emotion" Button component.
  </button>
  <a
    className="css-1x4026l-Link"
  >
    This uses the css prop
  </a>
</div>
`;
